client/src/components/Draftail/CommentableEditor/CommentableEditor.tsx

```diff
@@ -394,6 +398,35 @@ function forceResetEditorState(editorState: EditorState, replacementContent: Con
   return EditorState.acceptSelection(state, state.getSelection());
 }
 
+export function addCommentsToEditor(
+  contentState: ContentState,
+  comments: Comment[],
+  commentApp: CommentApp,
+  getAnnotation: () => Annotation
+) {
+  let newContentState = contentState;
+  comments.filter(comment => !comment.annotation).forEach((comment) => {
+    commentApp.updateAnnotation(getAnnotation(), comment.localId);
```

Comment:
Would firing off lots of redux actions on initial load cause a lot of re-renders?

Reply:
Yeah, this is something we could optimize